src/main/java/salesforce/api/AccountHelper.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+
+import salesforce.restclient.RestApi;
+import salesforce.entities.Account;
+import salesforce.utils.EndPoint;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
```

Comment:
imports are not necessary separate

please press ctrl + alt + O to the identation of the comments